src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ReloadableProject.cs

```diff
@@ -143,7 +153,13 @@ public Task Initialize()
                         // The only safe thing to do at this point is to reload the project in the solution
                         // TODO: should we have an additional return value here to indicate that the existing project could be in a bad
                         // state and the reload needs to happen without the user being able to block it?
-                        _telemetryService.Report("projectreload/reload-failed", "Project Reload Failure due to general fault", ex);
+                        var correlationEvent = _telemetryService.Report(ReloadFailedNFWPath, "Project Reload Failure due to general fault", ex);
+                        _telemetryService.PostOperation(
```

Comment:
We would like to know the failure rate. What does putting this in a using block get us over what I'm doing already, besides time duration? @srivatsn can correct, but I don't think we care about reload times, and wrapping everything in a using seems like it would make this more complicated than necessary.